src/commands/print.rs

```diff
@@ -19,8 +19,8 @@ pub struct GenerateCss {}
 
 impl GenerateCss {
     pub fn run(&self) -> Result<()> {
-        oranda::site::layout::css::build_css("dist")?;
-        tracing::info!("CSS placed in dist/oranda.css");
+        oranda_generate_css::build_css("oranda-css/dist")?;
```

Comment:
this shows up in enough random places that it should probably be some kind of const in oranda_generate_css...

Reply:
putting it in there would need us to publish a new version right now, which would break lockstep with oranda itself... kinda annoying